src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/probation/ProbationUnmergeEventListenerIntTest.kt

```diff
@@ -427,6 +427,14 @@ class ProbationUnmergeEventListenerIntTest : MessagingMultiNodeTestBase() {
       UNMERGE_MESSAGE_RECEIVED,
       mapOf("REACTIVATED_CRN" to reactivatedCrn, "UNMERGED_CRN" to unmergedCrn, "EVENT_TYPE" to OFFENDER_UNMERGED, "SOURCE_SYSTEM" to "DELIUS"),
     )
+    checkTelemetry(
+      CPR_RECORD_UNMERGED,
+      mapOf(
+        "REACTIVATED_CRN" to reactivatedCrn,
+        "UNMERGED_CRN" to unmergedCrn,
+        "SOURCE_SYSTEM" to "DELIUS",
+      ),
+    )
   }
 
   @Test
```

Comment:
This test fails frequently because the previous test hasn't finished processing the unmerge event yet, so it makes http calls which are no longer stubbed. The above telemetry check should sort it out